hyperwallet/utils/apiclient.py

```diff
@@ -107,7 +107,7 @@ def _makeRequest(
                 headers=headers,
                 params=params
             )
-        except:
+        except Exception:
```

Comment:
They might have left this part "catch-all" intentionally as it involves network and OS https://docs.python.org/3.6/library/exceptions.html#base-classes

Reply:
BaseException if not Simon, it's the python core one equivalent to this

Reply:
```suggestion
        except BaseException:
```